### PR TITLE
[REEF-323] Update README.md with build instructions

### PR DESCRIPTION
**Pull request by markusweimer**
JIRA:
  [REEF-323](https://issues.apache.org/jira/browse/REEF-323)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,77 @@
...
+
+To test, execute the following command thereafter:
+
+    vstest.console.exe .\lang\cs\bin\x64\Debug\Org.Apache.REEF.Tests\Org.Apache.REEF.Tests.dll /Platform:x64
 
+Additional Information
+----------------------
+More detailed documentation, including building from PowerShell and
+creating NuGets is available from the [project
+wiki](https://cwiki.apache.org/confluence/display/REEF/Home),
```
Comment (jwang98052) [inline README.md]: "project wiki" should be in the same line otherwise it would shown up as two lines.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,77 @@
...
+specifcially the [building
+instructions](https://cwiki.apache.org/confluence/display/REEF/Compiling+REEF).
```
Comment (jwang98052) [inline README.md]: "building instruction" shows as two lines in preview.

**Comment by markusweimer** [comment]
I did verify that the file renders correctly in GitHubs preview, e.g. here:

https://github.com/Microsoft-CISL/incubator-reef/blob/REEF-323/README.md

Hence, the question where to put the line break is taste. I can fix it if you want.

**Comment by jwang98052** [comment]
That's fine. It might be caused by my preview window. I will go ahead to merge it.

**Comment by bgchun** [comment]
Thanks, Markus!
It looks good.

On Thu, May 14, 2015 at 9:43 AM, asfgit <notifications@github.com> wrote:

> Closed #182 <https://github.com/apache/incubator-reef/pull/182> via
> 9b4b167
> <https://github.com/apache/incubator-reef/commit/9b4b167d29f2990bab4a6c0ecbc3dcab4dd7d7e8>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/182#event-304449063>.
>



-- 
Byung-Gon Chun

**Pull request closed without merging**